ocean_lib/data_provider/data_service_provider.py

```diff
@@ -103,7 +102,137 @@ def encrypt(
 
     @staticmethod
     @enforce_types
-    def sign_message(wallet: Wallet, msg: str) -> str:
+    def fileinfo(did: str, service_id: str, service_endpoint: str) -> Response:
```

Comment:
there's no fileinfo?!?! Why do we keep adding endpoints that most certainly already exist?

Reply:
There are only check functions which implement the fileinfo request inside them. There is not a specific method of fileinfo in ocean.py.